.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "extra/coredumper"]
```

Comment:
For some reason `PerconaFT` submodule is updated as part of this commit. Is this intentional?

Reply:
Accidental change, reverted.